src/main/java/practice/StreamPractice.java

```diff
@@ -1,80 +1,73 @@
 package practice;
 
-import java.util.Collections;
+import java.util.Arrays;
 import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 import model.Candidate;
 import model.Person;
 
 public class StreamPractice {
-    /**
-     * Given list of strings where each element contains 1+ numbers:
-     * input = {"5,30,100", "0,22,7", ...}
-     * return min integer value. One more thing - we're interested in even numbers.
-     * If there is no needed data throw RuntimeException with message
-     * "Can't get min value from list: < Here is our input 'numbers' >"
-     */
-    public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+
+    public int findMinEvenNumber(List<String> numbers) {   
+        return numbers.stream()
+                .flatMap(n -> Arrays.stream(n.split(",")))
```

Comment:
do not use the abbreviation
All magic numbers, strings, symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)
```suggestion
                .flatMap(line -> Arrays.stream(line.split(DELIMITER)))
```